.circleci/config.yml

```diff
@@ -0,0 +1,62 @@
+version: 2.1
+
+workflows:
+  ruby-tests:
+    jobs:
+      - unit-tests:
+          matrix:
+            parameters:
+              version: ["2", "2.1", "2.2", "2.3", "2.4", "2.5", "2.6.5", "2.7"]
+      - unit-tests-legacy:
+          matrix:
+            parameters:
+              version: ["kneip/ree-1.8.7-2012.02","ruby:1.9.3","circleci/jruby:9"]
+
+orbs:
+  ruby: circleci/ruby@0.1.2
+
+jobs:
+  unit-tests:
+    parallelism: 1
+    parameters:
+      version:
+        type: string
+    docker:
+      - image: circleci/ruby:<< parameters.version >>
+    steps:
+      - checkout
+      - run:
```

Comment:
Is the versions step still needed now that things are running with the correct version?

Reply:
yeah, I think so